tests/h/search/index_test.py

```diff
@@ -149,10 +153,10 @@ def test_you_can_make_aggregations_on_user_raw(self, factories, index, search):
 
         response = search.execute()
 
-        user_bucket_1 = next(bucket for bucket in response.aggregations.user_raw_terms.buckets
-                             if bucket["key"] == "acct:someone@example.com")
-        user_bucket_2 = next(bucket for bucket in response.aggregations.user_raw_terms.buckets
-                             if bucket["key"] == "acct:Someone@example.com")
+        user_bucket_1 = next(bucket for bucket in response.aggregations.user_raw_terms.buckets if
```

Comment:
These formatting changes appear to be unrelated to the other changes in this PR. Can you remove them from this PR?